pkgs/by-name/ay/ayano/package.nix

```diff
@@ -0,0 +1,26 @@
+{
+  buildGoModule,
+  fetchFromGitHub,
+  lib,
+}:
+buildGoModule rec {
+  pname = "ayano";
+  version = "0.0.10";
```

Comment:
Hi, ayano released `0.1.0` version in Sep 30 and I believe that the version could be updated, with:

```
sha256 = "sha256-W6kwPedeOXZGAUjRmFfNIMJW3vpiRZkRIs1q+GsvJcM=";
```

and

```
vendorHash = "sha256-6C4RYatXcMZxqNiw6nIs3dEdro2S2vL8F8c+t5QeXUs=";
```

Also ayano now requires Go 1.23, though it does not look like an issue in unstable ([as it already uses 1.23 now](https://github.com/NixOS/nixpkgs/blob/081cc2c431912460a96f4b34c3dd9f54eec9597c/pkgs/top-level/all-packages.nix#L11433)).